getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -3,93 +3,122 @@
 
 Creating a C# script
 --------------------
 
-After you successfully setup C# for Godot, you should see the following option when selecting ``Attach script`` in the context menu of a node in your scene:
+After you successfully set up C# for Godot, you should see the following option
+when selecting ``Attach script`` in the context menu of a node in your scene:
 
 .. image:: img/attachcsharpscript.png
 
-Note that while some specifics change, most of the things work the same when using C# for scripting.
-If you're new to Godot, you may want to peruse the tutorials on :ref:`doc_scripting` at this point.
-While some places in the documentation still lack C# examples, most things can be transferred easily from GDScript.
+Note that while some specifics change, most concepts work the same
+when using C# for scripting. If you're new to Godot, you may want to follow
+the tutorials on :ref:`doc_scripting` at this point.
+While some places in the documentation still lack C# examples, most concepts
+can be transferred easily from GDScript.
 
 Project setup and workflow
 --------------------------
 
-When you create the first C# script, Godot initializes the C# project files for your Godot project.
-This includes generating a C# solution (``.sln``) and a project (``.csproj``) file, as well as some utility files and folders (``.mono``, sometimes ``Properties``).
-All of these but ``.mono`` are important and should be kept in your version control system. ``.mono`` can be safely added to the ignore list of your VCS.
-When troubleshooting, it can sometimes help to delete the ``.mono`` folder and let it regenerate.
-
-Note that currently, there are some issues where Godot and the C# project don't stay in sync; if you delete, rename or move things, like scripts or nodes, they may no longer match up.
-In this case, it can help to edit the solution files manually.
-
-Example: If you created a script (e.g. ``Test.cs``) and delete it in Godot, the compilation will fail because now the missing file is still expected to be there by the CS project.
-For now, you can simply open up the ``.csproj`` file and look for the ``ItemGroup``, there should be a line included like the following:
+When you create the first C# script, Godot initializes the C# project files
+for your Godot project. This includes generating a C# solution (``.sln``)
+and a project file (``.csproj``), as well as some utility files and folders
+(``.mono``, sometimes ``Properties``).
+All of these but ``.mono`` are important and should be committed to your
+version control system. ``.mono`` can be safely added to the ignore list of your VCS.
+When troubleshooting, it can sometimes help to delete the ``.mono`` folder
+and let it regenerate.
+
+Note that currently, there are some issues where Godot and the C# project
+don't stay in sync; if you delete, rename or move things like scripts or nodes,
+they may no longer match up. In this case, it can help to edit
+the solution files manually.
```

Comment:
I will say this in a different way: "if you delete, rename or move a C# script, the change may not be reflected in the C# project file. In cases like this, it will require editing it manually."